hbp_knowledge/receptors/buckingham2009.bel

```diff
@@ -1203,22 +1296,27 @@ al., 2007) through the activation of nAChRs. It therefore
 seems that activation of nAChRs shifts the balance of
 APP processing away from beta-amyloidogenic to soluble
 APP production."
-
+SET Confidence = "Low"
 SET CellLine="PC12"
 
-p(HGNC:CHRNA7) -> a(CHEBI:nicotine)
-a(CHEBI:nicotine) -> sec(a(HBP:"sAPP-beta"))
+a(CHEBI:nicotine) -> p(HGNC:CHRNA7)
+#nicotine stimulates CHRNA7 and not vice versa
+p(HGNC:CHRNA7) -> sec(a(HBP:"sAPP-beta"))
 a(HBP:"sAPP-beta") -| act(a(CHEBI:"amyloid-beta"))
 UNSET CellLine
 
 a(CHEBI:nicotine) -| a(HBP:"sAPP-gamma")
+#can we write here also that nicotine stimulates cholinergic receptors, which in turn lowers aAPP-gamma? Nicotine does not do it directly after all.
```

Comment:
if you use the "-|" instead of "=|" it is not a direct interaction. And as the activation of receptors due to nicotine leads to the reduction of sAPP-gamma, this statement would be correct in my opinion